packages/manager/src/features/Linodes/HighPerformanceVolumeIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import BoltIcon from '@mui/icons-material/Bolt';
+import { IconButton, Tooltip } from '@mui/material';
```

Comment:
I didn't catch this before -- we're going to want to use our components from the`'@linode/ui` package here, not the imports from Material UI.

```suggestion
import { IconButton, Tooltip } from '@linode/ui';
```